src/components/ui/icons/UpAndDownCaret/index.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import Svg, { SvgProps } from '../Svg';
+
+const UpAndDownCaret: React.SFC<SvgProps> = (props) => (
+  <Svg {...props}>
+    <path d="M8.824 9.88c-.21.18-.526.154-.705-.056-.159-.187-.156-.457-.006-.64l.063-.065 3.523-3c.165-.14.397-.156.577-.05l.074.052 3.477 3c.209.18.232.497.052.706-.16.185-.428.224-.632.104l-.074-.052-3.151-2.72-3.198 2.722zM15.176 14.12c.21-.18.526-.154.705.056.159.187.157.457.006.64l-.063.065-3.523 3c-.165.14-.397.156-.577.05l-.074-.052-3.477-3c-.209-.18-.232-.497-.052-.706.16-.185.428-.224.632-.104l.074.052 3.151 2.72 3.198-2.722z" />
+  </Svg>
+);
+
+UpAndDownCaret.displayName = 'Up and Down Caret';
```

Comment:
Maybe I'm wrong, but I don't think the displayName prop should have spaces in it.

Reply:
Thanks Brad Will update since I just looked at some and they all one word.